boltdesignsystem/bolt
# DS 833 list element

## Jira

https://pegadigitalit.atlassian.net/browse/DS-833

## Summary

A list element is created that will replace the list component.

## Details
- a new list element doesn't use JS
- all props will be CSS classes that will modify the markup to adjust the display appropriately
- the new element has a similar schema as the old list component
  -- the `flex` option was deleted from the `display` prop,
  -- the `inline` and the `inline@breakpoint` props were renamed to `horizontal` and `horizontal@breakpoint`,
  --  the `block` prop was renamed to `vertical`,


## How to test

Pull the branch. In the docs, there is various example of the list variations. Make sure that the new element behaves like the old list component.

## Release notes

There is a new List Element and The List Component was marked as deprecated.
